### PR TITLE
Randomize chores model method

### LINKED ISSUE #31: Write method for assigning chores evenly (creating MemberChore entries)

### PR DESCRIPTION
**Pull request by aj-bailey**
# Description

This PR implements the following:
- #randomize_chores Household instance method and testing
- Household and Member association tests

<br>

## Type of Changes:

- [ ] Bug fix
- [x] New feature
- [ ] Refactoring
- [x] Testing

<br>
<br>

# Related Issue(s):

- [x]  closes #31 

<br>

## Other Considerations 

- [x] I have linted my code locally before submission
- [x] This change requires a documentation update

### CODE REVIEW

**Review by Bphayes1200 — APPROVED** [review]
Looks good, Thanks.

File: spec/graphql/mutations/randomize_chores_spec.rb
Comment (MoxieMax) [inline spec/graphql/mutations/randomize_chores_spec.rb:1]: I was surprised there were no tests here, but @hamouj let me know that this was started before you switched to working on the model.

File: spec/models/household_spec.rb [hunk 1]
```diff
@@ -4,5 +4,95 @@
...
+      let!(:chore9) { create(:chore, name: "chore9", household:, duration: 45) }
+      let!(:chore10) { create(:chore, name: "chore10", household: household2, duration: 45) }
+      let!(:member_chore10) { create(:member_chore, member: member4, chore: chore10) }
+
+      it "creates member chores that are evenly distributed" do
+        household.randomize_chores
+
+        Household.find(household.id).members.each do |member|
+          expect(total_chore_duration(member)).to be >= 60
+          expect(total_chore_duration(member)).to be <= 115
```
Comment (MoxieMax) [inline spec/models/household_spec.rb:37]: I've never seen testing done this way for multiple objects, it seems way more efficient than writing the testing for each individually

**Review by hamouj — APPROVED** [review]
Noted the creation of the randomize chores method along with helper methods for the household model.
We may look at refactoring to see if we can un-nest some of the iterations in the future.

Model tests make sense; we may consider adding in tests for chores with higher durations to fully cover possibilities (accepted values are 15-240).

Nice job with linting--LGTM!

**Review by MoxieMax — APPROVED** [review]
The methods in the household are complex, I'm excited to see them in action.

**Pull request merged**